src/cirrus/builtins/functions/update-state/lambda_function.py

```diff
@@ -112,15 +110,16 @@ def workflow_completed(execution: Execution) -> None:
     # trying the sns publish, but I could see it the other
     # way too. If we have issues here we might want to consider
     # a different order/behavior (fail on error or something?).
-    statedb.set_completed(execution.input["id"], execution_arn=execution.arn)
+    wf_event_manager.succeeded(execution.input["id"], execution_arn=execution.arn)
```

Comment:
maybe this method should take `execution.input` as it's parameter to be consistent with all of the others?